src/screens/Menu/Menu/Menu.js

```diff
@@ -111,15 +120,19 @@ const Menu = () => {
         />
         <MenuItem title={t('item.addressBook')} icon="contacts" onPress={goToInviteFriends} />
         {enoughPlrBalance ? (
-          <MenuItem title={t('item.liveChatSupport')} icon="message" onPress={goToSupportConversations} />
-        ) : (
+          <MenuItem title={t('item.liveChatSupport')} icon="message" onPress={goToEmailSupport} />
+        ) : null}
```

Comment:
Just one thing, if we are giving email support, if the user has enough PLR balance or not so we can put default condition instead of separate conditions for both enoughPlrBalance and !enoughPlrBalance
<MenuItem title={t('item.liveChatSupport')} icon="message" onPress={goToEmailSupport} />